oc.po

```diff
@@ -0,0 +1,54 @@
+# Occitan translation.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the snapd-desktop-integration package.
+# Quentin PAGÈS, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: snapd-desktop-integration\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-10-24 15:38+0200\n"
+"PO-Revision-Date: 2023-02-28 14:51+0100\n"
+"Last-Translator: Quentin PAGÈS\n"
+"Language-Team: \n"
+"Language: oc\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/main.c:59 src/main.c:65 src/main.c:88
+msgid "Installing missing theme snaps:"
+msgstr "Installacion de snaps de tèma mancants :"
```

Comment:
That blank space between the colon... Is a typo?

Reply:
no, it's like in French, we put a no-break space before :  ! and ?